src/Providers/OEmbed.php

```diff
@@ -104,6 +104,23 @@ public function getType()
     /**
      * {@inheritdoc}
      */
+    public function getTags()
+    {
+        if ($this->bag->has('meta')) {
+            //it means we are using iframe.ly api
+            $meta_bag=$this->bag->get('meta');
+            if (isset($meta_bag['keywords'])) {
+                 return array_map('trim', explode(',',$meta_bag['keywords']));
+            }
+        }
```

Comment:
FYI, you can use directly `$this->bag->has('meta[keywords]')` and `$this->bag->get('meta[keywords]')` (shorter and simpler)